lib/deployinator/templates/layout.mustache

```diff
@@ -53,6 +53,7 @@ _  __  / _  _ \___  __ \__  / _  __ \__  / / /__  / __  __ \_  __ `/_  __/_  __
                        <span class="label">Stack: </span>
                        <span>
                            <select name='stacks' id='stacks'>
+                           <option></option>
```

Comment:
Perhaps we should put some default text in here as well, like "Choose a stack" or just "Choose..."